getting_hired/preparing_for_job_search/professional_networking.md

```diff
@@ -0,0 +1,82 @@
+### Introduction
+
+A robust professional network is an irreplaceable asset on your journey to becoming a web developer, and beyond. As social beings we are wired to connect with others, receive help, and give help. This is no different in the software ecosystem where countless communities take part in the mutually beneficial transactions of networking.
+
+One significant benefit of your network is that it might help land you your first job in software. This is not a guarantee, but it is the lived experience of many developers. As with all things, you get back what you put in, so the more developed your network is, the better your odds it will provide for you. 
+
+In this lesson we cover what a professional network is, how it might benefit you, and some strategies to consider when to building one. Making genuine connections with people takes time, however, so we recommend you start investing time and effort sooner than later.
+
+### What is a professional network?
+
+Your professional network is not the people you follow on social media, nor is it necessarily your friends. It is the people with whom you have made a genuine connection, whether in-person or online, because you have a mutual interest and can gain more value through being connected than you would otherwise have alone. These are the people you can ask for help, and in turn help out.
```

Comment:
and in turn help **_you_** out.

Reply:
Ah I meant they can help you, and you can help them. I reworded slightly.